doc/cards.xsd

```diff
@@ -23,54 +32,37 @@
-                                </xs:element>
-                                <xs:element type="xs:string" name="color" minOccurs="0" maxOccurs="5" />
-                                <xs:element type="xs:string" name="manacost" maxOccurs="1" />
-                                <xs:element type="xs:string" name="type" minOccurs="1" maxOccurs="1" />
-                                <xs:element type="xs:string" name="pt" minOccurs="0" maxOccurs="1" />
-                                <xs:element type="xs:integer" name="tablerow" maxOccurs="1" />
-                                <xs:element type="xs:string" name="text" maxOccurs="1" />
-                                <xs:element type="xs:boolean" name="cipt" minOccurs="0" maxOccurs="1" />
-                                <xs:element type="xs:string" name="loyalty" minOccurs="0"  maxOccurs="1" />
-                                <xs:element type="xs:string" name="cmc" minOccurs="1" maxOccurs="1" />
-                                <xs:element type="xs:boolean" name="upsidedown" minOccurs="0" maxOccurs="1" />
-                                <xs:element type="xs:boolean" name="token" minOccurs="0" maxOccurs="1" />
-                                <xs:element  name="related" minOccurs="0" maxOccurs="unbounded">
-                                    <xs:complexType>
-                                        <xs:simpleContent>
-                                            <xs:extension base="xs:string">
-                                                <xs:attribute type="xs:string" name="count" use="optional"/>
-                                                <xs:attribute type="xs:string" name="exclude" use="optional"/>
-                                                <xs:attribute type="xs:string" name="attach" use="optional"/>
-                                            </xs:extension>
-                                        </xs:simpleContent>
-                                    </xs:complexType>
-                                </xs:element>
-                                <xs:element  name="reverse-related" minOccurs="0" maxOccurs="unbounded">
-                                    <xs:complexType>
-                                        <xs:simpleContent>
-                                            <xs:extension base="xs:string">
-                                                <xs:attribute type="xs:string" name="count" use="optional"/>
-                                                <xs:attribute type="xs:string" name="exclude" use="optional"/>
-                                                <xs:attribute type="xs:string" name="attach" use="optional"/>
-                                            </xs:extension>
-                                        </xs:simpleContent>
-                                    </xs:complexType>
-                                </xs:element>
-                            </xs:choice>
+                            <xs:sequence>
```

Comment:
XML 1.1 allows an `<xs:all>` that isn't restricted to 0 or 1 instances for every element.